target/linux/ath79/dts/qca9531_comfast_cf-e355ac-v2.dts

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "comfast,cf-e355ac-v2", "qca,qca9531";
+	model = "COMFAST CF-E355AC V2";
+
+	aliases {
+		serial0 = &uart;
+		label-mac-device = &eth1;
```

Comment:
Please respond to my earlier comment. This is probably wrong, and the label MAC address should be that of WAN, i.e. 0x0 +0?

Reply:
No, eth0 is LAN, eth1 is WAN:
```
ip addr
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel master br-lan state UP qlen 1000
    link/ether 20:0d:b0:7a:e6:c1 brd ff:ff:ff:ff:ff:ff
3: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP qlen 1000
    link/ether 20:0d:b0:7a:e6:c0 brd ff:ff:ff:ff:ff:ff
    inet 10.42.0.77/24 brd 10.42.0.255 scope global eth1
```

Reply:
eth0 is probably NOT &eth0, that's what I mean, since the DT names and runtime names are probably swapped!